app.py

```diff
@@ -26,11 +28,18 @@ def init_app(self):
         # make the util methods available via the app instance
         self.util = tk_multi_publish2.util
 
+        display_name = self.get_setting("display_name")
+        # "Publish Render" ---> publish_render
+        command_name = display_name.lower().replace(" ", "_")
+
+        if command_name.endswith("..."):
```

Comment:
I think we want to account for this in a more generic way, like the suggestion above. I suggest we remove this check.